reactioncommerce/reaction
# Add validation for required fields on PDP

*Resolves*
#2225 

*Testing*
- Login as admin
- Create a product
- Add a variant option.
- Fill all fields apart from product title, variant details label and variant options label.
- Click publish
- A toast message should show what fields are required.
- Now fill in all field required, then click publish
- The changes should be successfully published.
